binaries/conda/build_packages.sh

```diff
@@ -4,12 +4,18 @@ set -eou pipefail
 
 PYTHON_VERSIONS="3.6 3.7 3.8"
 PKGS="torchserve torch-model-archiver"
+TORCHSERVE_VERSION=`cat ../../ts/version.txt`
+TORCH_MODEL_ARCHIVER_VERSION=`cat ../../model-archiver/model_archiver/version.txt`
 
 for pkg in ${PKGS}; do
+    PKG_VERSION=$(echo $pkg | tr 'a-z' 'A-Z' | tr '-' '_')_VERSION
+    cat $pkg/meta.yaml.tmpl | sed -e "s/{version}/$PKG_VERSION/" >> $pkg/meta.yaml
```

Comment:
For this we can actually for-go having to use a template at all and instead prefer to use environment variables here.

For reference, this is how we do it for our upstream `pytorch` conda packages:

```yaml
package:
  name: pytorch
  version: "{{ environ.get('PYTORCH_BUILD_VERSION') }}"
```

https://github.com/pytorch/builder/blob/191b23db356f04b7501cfc818c45a1a4db66d7db/conda/pytorch-nightly/meta.yaml#L1-L3

Reply:
Done.